spec/services/form1010cg/service_spec.rb

```diff
@@ -46,6 +46,31 @@
 
       expect(service.claim).to eq(claim)
     end
+
```

Comment:
I would rethink this test, It shouldn't matter what `carma_client` is. take a look at [this](https://github.com/department-of-veterans-affairs/vets-api/blob/a21cc416ee41686a6c7431ab34d0924229dcc1d9/modules/facilities_api/spec/services/facilities_api/v1/ppms/client_spec.rb#L20-L24). This file runs all of the tests with the flag off and on. I do have to branch for how things behave differently. I know it's more test code to maintain but I think its worth it since Feature flags should be temporary.